src/hotspot/share/oops/instanceKlass.cpp

```diff
@@ -1387,6 +1387,12 @@ instanceOop InstanceKlass::allocate_instance(TRAPS) {
   return i;
 }
 
+instanceOop InstanceKlass::allocate_instance(InstanceKlass* ik, TRAPS) {
```

Comment:
Should this have a null check on ik, like in the inlined function that it replaced?